packages/callout-block/tsconfig.json

```
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```

```suggestion
        "types": ["cypress"],
```

Also this shouldn't be needed on that block